tests/components/xiaomi/test_device_tracker.py

```diff
@@ -154,17 +154,17 @@ def raise_for_status(self):
 async def test_config(xiaomi_mock, hass: HomeAssistant) -> None:
     """Testing minimal configuration."""
     config = {
-        DOMAIN: xiaomi.PLATFORM_SCHEMA(
+        DEVICE_TRACKER_DOMAIN: xiaomi.PLATFORM_SCHEMA(
             {
-                CONF_PLATFORM: xiaomi.DOMAIN,
+                CONF_PLATFORM: DEVICE_TRACKER_DOMAIN,
```

Comment:
`xiaomi.DOMAIN` would refer to `homeassistant.components.xiaomi.device_tracker.DOMAIN`, which in turns referred to `homeassistant.components.device_tracker.DOMAIN`
This has now been simplified to `DEVICE_TRACKER_DOMAIN`

Maybe the intention was to use `"xiaomi"` and not `"device_tracker"` but I decided to match the existing string value.